### ISSUE TITLE
render and nose problems

### ISSUE THREAD

**Issue by flynnamy** [apchenstu/Facial_Details_Synthesis#23]
Hi,Could I just do render_texture?Because I find that it only depends on textureRender.exe and image path.But when I execute render_texture function, it can only get result.isomap which is not right.Could you please tell me the reason? On the other hand, the result of nose is not like shows in paper.I know it has occlusions,how do you solve this and make it show like paper's result? One is  result.isomap which only did render_texture and the other has nose problem like following. Looking forward to hearing from you! @apchenstu 
![result isomap](https://user-images.githubusercontent.com/20398890/70703472-2de43e00-1d0b-11ea-9674-fd7c8ba8397c.png)
![image](https://user-images.githubusercontent.com/20398890/70703528-4b190c80-1d0b-11ea-880e-6fc4a6eacd4e.png)

**Comment by apchenstu** [comment]
Hi, for the nose issue, please refer to [this](https://github.com/apchenstu/Facial_Details_Synthesis/issues/12#issuecomment-528824360). Thanks.

**Comment by flynnamy** [comment]
@apchenstu Hi,how about first question?Could I just do render_texture?Because I find that it only depends on textureRender.exe and image path.But when I execute render_texture function, it can only get result.isomap which is not right.Could you please tell me the reason?

**Comment by apchenstu** [comment]
Yes, texture render is independent with other programs, you can simply run it via command line. 
Could you provide me more information? For example, your input command line, input files, and outputs or error message.

**Comment by XIAO-RUI-RUI** [comment]
> @apchenstu Hi,how about first question?Could I just do render_texture?Because I find that it only depends on textureRender.exe and image path.But when I execute render_texture function, it can only get result.isomap which is not right.Could you please tell me the reason?

Try to copy `.\src\textureRender\src\shaders` to `.\src\textureRender\build\Release`, maybe the .\Release\textureRender.exe can not find .\shader.
